.openshift-ci/pre_tests.py

```diff
@@ -56,3 +57,15 @@ def run(self):
             check=True,
             timeout=PreSystemTests.START_PREFETCH_TIMEOUT,
         )
+
+
+class CollectionPreTest:
+    """
+    CollectionPreTest - allows finer control over collection method
+    for individual tests
+    """
+    def __init__(self, method):
+        self._collection_method = method or "NO_COLLECTION"
```

Comment:
Does NO_COLLECTION not start collector container at all? From the [test](https://prow.ci.openshift.org/view/gs/test-platform-results/pr-logs/pull/stackrox_stackrox/11946/pull-ci-stackrox-stackrox-master-gke-version-compatibility-tests/1812821859058585600) logs it seems so.

Reply:
Yeah, it'll not start any Collectors at all. The collector pods will still exist, but only containing the compliance container